File: 2021_03_Problem01.py
n=int(input())
string=input()
timeline=[0]*24
string=string.split( )
string=[int(x) for x in string]
for i in range(0,len(string),2):
    for j in range(string[i],string[i+1]+1):
        timeline[j-1]+=1
print(max(timeline))
    

File: 2021_03_Problem01.cpp
#include <iostream>
using namespace std;
int main() {
    int n;
    cin>>n;
    int s[n],d[n];
    int timeline[24];
    for(int i=0;i<24;i++)timeline[i]=0;
    for(int i=0;i<n;i++){
        cin>>s[i]>>d[i];
        for(int j=s[i]-1;j<=d[i]-1;j++){
            timeline[j]++;
        }
    }
    int max=0;
    for(int i=0;i<24;i++){
        if(timeline[i]>=max)max=timeline[i];
    }
    cout<<max<<endl;
    return 0;
}
